src/pulse3D/excel_writer.py

```diff
@@ -275,14 +275,16 @@ def write_xlsx(
     file_suffix = "full" if is_full_analysis else f"{start_time}-{end_time}"
     output_file_name = f"{input_file_name_no_ext}_{file_suffix}.xlsx"
 
-    if first_wf.stiffness_override:
-        # reverse dict to use the stiffness factor as a key and get the label value
-        post_stiffness_factor = {v: k for k, v in POST_STIFFNESS_LABEL_TO_FACTOR.items()}[
-            first_wf.stiffness_factor
-        ]
+    if "xlsx" not in plate_recording.path:
```

Comment:
I think it would be better to just set `stiffness_override` to `None` for an xlsx file